warehouse/templates/email/new_email_added/body.txt

```diff
@@ -0,0 +1,20 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "email/_base/body.txt" %}
+
+{% block content %}
+{% trans email_address=email_address %}Your new email '{{ email_address }}' has been added.{% endtrans %}
+
+{{ message }}
```

Comment:
Same note about `{{ message }}` -- it doesn't appear in your rendering body, so this will produce an error.